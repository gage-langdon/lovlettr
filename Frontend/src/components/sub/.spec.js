import React from 'react';
import { shallow } from 'enzyme';
//component
import Sub from './index';

describe('Sub component', () => {
  it('Will render with just text passed in', () => {
    const wrap = shallow(<Sub text="test" />);
    expect(wrap).toMatchSnapshot();
  });
});
