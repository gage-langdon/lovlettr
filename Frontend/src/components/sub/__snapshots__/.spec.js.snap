// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sub component Will render with just text passed in 1`] = `
<sub
  className=""
  hidden={false}
>
  test
</sub>
`;
